draft-ietf-sipcore-sip-push.xml

```diff
@@ -50,78 +50,90 @@
-        Push notifications might contain payload data, depending on the application. An application 
-        can request that a push notification is sent to a single user application, or to multiple
-        user applications.
+        In order to save resources such as battery life, some devices (especially mobile devices)
+        and operating systems will suspend an application that is not in use. A suspended application 
+        might not be able to wake itself with internal timers and might not be awakened by incoming 
+        network traffic. In such an environment, a Push Notification Service (PNS) is used to wake 
+        the application. A PNS is a service that sends messages requested by other applications 
+        to a user application in order to wake the user application. These messages are called push 
+        notifications.  Push notifications might contain payload data, depending on the application.  
+        An application can request that a push notification is sent to a single user application or 
+        to multiple user applications.
       </t>
       <t>  
-        Typically each operating system uses a dedicated PNS. 
-        For example, Apple iOS devices use the Apple Push Notification service (APNs)
-        while Android devices use the Firebase Cloud Messaging (FCM) service.
+        Typically each operating system uses a dedicated PNS. Different PNSs exist today. 
+        Some are based on the standardized mechanism defined in <xref target="RFC8030"/>, while 
+        others are proprietary. For example, Apple iOS devices use the Apple Push Notification 
+        service (APNs) while Android devices use the Firebase Cloud Messaging (FCM) service.
+        Each PNS uses PNS-specific terminology and function names. The terminology in this 
+        document is meant to be PNS-independent. If the PNS is based on <xref target="RFC8030"/>, 
+        the SIP proxy takes the role of the application server.
       </t>
       <t>
-        Because of the restrictions above, Session Initiation Protocol (SIP) User Agents (UAs)
-        <xref target="RFC3261"/> can not be awoken, in order to send binding-refresh SIP REGISTER
-        requests and to receive incoming SIP requests, without using a PNS to wake the UA
-        in order to perform those functions. 
+        When a Session Initiation Protocol (SIP) User Agent (UA)<xref target="RFC3261"/> is 
+        suspended in such an environment, it is unable to send binding-refresh SIP REGISTER 
+        requests, unable to receive incoming SIP requests, and might not be able to use internal 
+        timers to wake itself. A suspended UA will not be able to maintain connections e.g.,
+        using the SIP Outbound Mechanism <xref target="RFC5626"/> because it cannot send periodic 
+        keep-alive messages. A PNS is needed to wake the SIP UA so that the UA can perform these 
+        functions.
       </t>
```

Comment:
I don't think we should talk about Outbound. We have agreed that Outbound support of SIP Push would be documented in a separate draft, because it would require more work than simply sending push notifications in order to trigger keep-alives.

Reply:
The sentence about Outbound was added to -05 due to a comment from the WGLC [1]. There was a thread discussing Outbound [2], and also some threads discussing moving it to another draft [3] [4]. I think the sentence mentioning Outbound is ok since it is just used as an example. Would it be useful to add a sentence here about how using Outbound with sip-push is out of scope and will be covered in another document? 

[1] https://mailarchive.ietf.org/arch/msg/sipcore/nLVgRd6pps7-DEaZe8YITjUTA8Y
[2] https://mailarchive.ietf.org/arch/msg/sipcore/mHFu3ZyPNOcrAZuJWqSc2fWv1ok
[3] https://mailarchive.ietf.org/arch/msg/sipcore/8OkrjthsRfrUi1zwULj5kzC68ao
[4] https://mailarchive.ietf.org/arch/msg/sipcore/PC6y73rOxjeW8XupVZRQjkJLVIs

Reply:
You are correct, the reference to Outbound was previously added. I withdraw that comment.

I guess we can add a note saying that the details of how to use SIP Push together with Outbound is outside the scope of the document.